includes/templates/docs/shortcode-loop.php

```diff
@@ -0,0 +1,41 @@
+<?php
+// Set defaults from arguments passed to template.
+// $args won't exist in WP <5.5
```

Comment:
I can't 100% follow the logic here, but I assume this means that the defaults below ('show_date' = `false` etc) will ensure that this doesn't break in WP < 5.5?